frontend/src/components/common/private-route/private-route.tsx

```diff
@@ -22,6 +22,38 @@ const PrivateRoute: FC<Props> = ({
     return <Navigate to={redirectTo} />;
   }
 
+  const { pathname } = useLocation();
+
+  let requiredPermission: string;
+
+  switch (pathname) {
+    case AppRoute.EAM: {
+      requiredPermission = 'manage-eam';
+      break;
+    }
+    case AppRoute.BS: {
+      requiredPermission = 'manage-bs';
+      break;
+    }
+    case AppRoute.SC: {
+      requiredPermission = 'manage-sc';
+      break;
+    }
+    case AppRoute.SLC: {
+      requiredPermission = 'manage-slc';
+      break;
+    }
+    default: {
+      requiredPermission = '';
+    }
+  }
+
+  if (requiredPermission) {
+    if (!user?.permissions.includes(requiredPermission)) {
+      return <Navigate to={AppRoute.ROOT} />;
+    }
+  }
```

Comment:
I think you can avoid 2 if here.
```suggestion
  if (requiredPermission && !user?.permissions.includes(requiredPermission)) {
    return <Navigate to={AppRoute.ROOT} />;
  }
```